src/RSocketResponder.cpp

```diff
@@ -7,12 +7,14 @@ namespace rsocket {
 
 yarpl::Reference<yarpl::single::Single<rsocket::Payload>>
 RSocketResponder::handleRequestResponse(rsocket::Payload request, rsocket::StreamId streamId) {
+  VLOG(4) << "handleRequestResponse not implemented.";
```

Comment:
it just makes me think , how much are these strings going to increase the size of the binary even if the log level is skipping the log statements...